src/components/Api.js

```diff
@@ -0,0 +1,142 @@
+class Api {
+  constructor({ baseUrl }) {
+    this._baseUrl = baseUrl;
+    this._headers = {
+      authorization: "6d7ae9eb-8869-410f-b35c-8e1dcc44f04b",
+      "Content-Type": "application/json",
+    };
+  }
+  //GET https://around-api.en.tripleten-services.com/v1/cards
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      method: "GET",
+      headers: this._headers,
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject("Error");
+      })
+      .catch((err) => {
```

Comment:
Needs correction: Errors should be handled outside of the api class in `index.js`. Now `catch` block is placed in the middle of the chain, so the rest of the chain won't know anything about the error — but it should because we need to update the data only in case of success

The api call in `index.js` should look like this:

```js
// show loading
api.
  .then(...)
  .catch(...)
  .finally(() => {
    // hide loading
  })
```